pkgs/by-name/op/openmpi/package.nix

```
@@ -83,6 +71,7 @@ stdenv.mkDerivation (finalAttrs: {
     USER = "nixbld";
     HOSTNAME = "localhost";
     SOURCE_DATE_EPOCH = "0";
+    NIX_CFLAGS_COMPILE = "-Wno-error=int-conversion";
   };
```

Hmm maybe separate this so that the reproducibility related comment above will be more accurate?

```suggestion
  };
  # TODO: Comment why is that needed?
  env.NIX_CFLAGS_COMPILE = "-Wno-error=int-conversion";
```

---

I got a little tired of adding that comment (nixpkgs is now littered with these compile flag statements). It is needed because of gcc-14, which has turned a bunch of different warnings into errors.